tools/integration_tests/setup/setup.go

```diff
@@ -57,14 +56,6 @@ func TestDir() string {
 	return testDir
 }
 
-func SetTmpDir(tmpDirValue string) {
```

Comment:
I see this function is deleted but its usages are not. Is it the case that this setter is not used anywhere?

Reply:
There are no uses remains for this function